package/base-files/files/sbin/sysupgrade

```diff
@@ -390,7 +390,7 @@ json_get_var valid "valid"
 }
 
 if [ -n "$CONF_IMAGE" ]; then
-	case "$(get_magic_word $CONF_IMAGE cat)" in
+	case "$(get_magic_word "$CONF_IMAGE" cat)" in
```

Comment:
An example of a broken configuration that trigger this?

Reply:
CONF_IMAGE is a user-provided value (from arg "-f"). So, without the quote, something like `sysupgrade -f "my backup.tgz" openwrt.img` would break. Luci is not affected as you cannot "flash with restore".